chia/plotting/plot_tools.py

```diff
@@ -159,25 +160,27 @@ def load_plots(
     if match_str is not None:
         log.info(f'Only loading plots that contain "{match_str}" in the file or directory name')
 
-    for filename in all_filenames:
+    def process_file(filename):
```

Comment:
Please use type annotations, we use these in the codebase to decrease bugs and increase development speed. This would be:
`def process_file(filename: Path) -> None:`